NuRadioReco/detector/antennapattern.py

```diff
@@ -103,7 +97,7 @@ def get_group_delay(vector_effective_length, df):
     return -np.diff(np.unwrap(np.angle(vector_effective_length))) / df / units.ns / 2 / np.pi
 
 
-def parse_WIPLD_file(ad1, ra1, orientation, gen_num=1, s_paramateres=[1, 1]):
+def parse_WIPLD_file(ad1, ra1, orientation, gen_num=1, s_parameters=None):
```

Comment:
why is this being changed by the linter

Reply:
I had to look this complaint up, too: If the default parameter of a function is something like a list or a dict, the list will be created the first time the function is called. Any function calls after that don't create a new list, but reference the first one. So if anything causes that list to change, the function will be called with a different default value. Therefore, Linters complain about something like this.
In this case, it's probably not too bad, since the parameters should not change, but better safe than sorry.

Reply:
yes that makes sense.